.gitignore

```diff
@@ -141,7 +141,6 @@ docs/site/
 
 # addon modules
 goldstone/static/addons/
-addon-requirements.txt
```

Comment:
This file has been moved to the add-ons application directory, and no longer exists here.  And its content has changed.  See below.